contrib/miniupnpc/template.py

```diff
@@ -0,0 +1,24 @@
+pkgname = "miniupnpc"
+pkgver = "2.2.8"
+pkgrel = 0
+build_style = "cmake"
+configure_args = ["-DUPNPC_BUILD_SAMPLE=OFF"]
+hostmakedepends = ["cmake", "ninja", "pkgconf"]
+pkgdesc = "UPnP client library"
+maintainer = "Erica Z <zerica@callcc.eu>"
+license = "BSD-3-Clause"
+url = "http://miniupnp.free.fr"
+source = f"{url}/files/miniupnpc-{pkgver}.tar.gz"
+sha256 = "05b929679091b9921b6b6c1f25e39e4c8d1f4d46c8feb55a412aa697aee03a93"
+tool_flags = {"CFLAGS": ["-fPIC"]}
```

Comment:
use the cmake CMAKE_POSITION_INDEPENDENT_CODE=ON instead

also what error do you get without this? obs also has this issue but is a bit too big to debug, i wonder why cmake is not just passing this by default